### PR TITLE
Redesign for Auctions#index view

### PR DESCRIPTION
**Pull request by jessieay**
* Rebased / updated version of https://github.com/18F/micropurchase/pull/602

### CODE REVIEW

**Comment by jessieay** [comment]
@gemfarmer I think there are a good number of code climate css violations here. I cleaned up the view logic so once  CC is taken care of this should be close to a merge!

**Comment by harrisj** [comment]
Yeah, this branch will have to be rebased too it looks like, but if you can also look at the violations...

**Comment by gemfarmer** [comment]
@jessieay Does this replace #602? I'm confused

**Comment by jessieay** [comment]
Yes, it replaces it. I wanted to keep your branch intact in case I
overwrote something during rebase. :)

**Comment by gemfarmer** [comment]
@jessieay gotcha!  I'm going to rebase and solve these scss issues now.

**Comment by gemfarmer** [comment]
@jessieay This should be good to merge. _fingers crossed_ the checks pass!

File: app/view_models/auction_list_item.rb [hunk 1]
```diff
@@ -43,92 +43,130 @@ def eligibility_label
...
-    end
+  def project_label
+    auction.github_repo.sub('https://github.com/', '')
   end
 
   def winning_bid_partial
+    if over? && !auction.bids.any?
+      'auctions/no_bids'
+    elsif auction.type == 'single_bid'
+      winning_single_bid_partial
```
Comment (jessieay) [inline app/view_models/auction_list_item.rb]: by going to these other methods we are making the logic a bit more complicated -- can we put it all pack into one method? I know it' SUPER long, but it makes it easier to understand the branching

Comment (gemfarmer) [inline app/view_models/auction_list_item.rb]: Definitely. I only did this because codeclimate was saying it was "too complex".

Comment (jessieay) [inline app/view_models/auction_list_item.rb]: if code climate was yelling at you for having a long conditional that is OK with me -- the point of getting this conditional on one place is to see all of the branching and extract new objects.

Comment (gemfarmer) [inline app/view_models/auction_list_item.rb]: I reverted it :shrug: It seems like a delicate balance. Oh the narrow lines we walk!

**Comment by jessieay** [comment]
Sorry I made you fail code climate! I think this is ready to merge once we finish our small discussion about ribbons in the slack design channel

**Comment by gemfarmer** [comment]
@jessieay ok new code pushed!

**Pull request merged**